src/tools/env.c

```diff
@@ -548,6 +548,7 @@ static kh_mapping_entry_t* mapping_entries = NULL;
 
 void RecordEnvMappings(uintptr_t addr, size_t length, int fd)
 {
+    if (!box64env_entries) return;
```

Comment:
Oh now, not like that. We don't want to remove the file mapped trcking for this reason. Because file map will have much more than just profile later on!

Reply:
You mean the aot cache? okay...

Reply:
Yeah, that. But before that, the dynarec memory...

Reply:
is it okay now?

Reply:
yep. just waiting for the CI.